prow/plugins/assign/assign.go

```diff
@@ -188,7 +188,7 @@ type handler struct {
 func newAssignHandler(e github.GenericCommentEvent, gc githubClient, log *logrus.Entry) *handler {
 	org := e.Repo.Owner.Login
 	addFailureResponse := func(mu github.MissingUsers) string {
-		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and repo collaborators can be assigned and that issues/PRs can only have 10 assignees at the same time.\nFor more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/#issue-assignment-in-github)", strings.Join(mu.Users, ", "), org, org)
+		return fmt.Sprintf("GitHub didn't allow me to assign the following users: %s.\n\nNote that only [%s members](https://github.com/orgs/%s/people) and the people should have commented on the issues/PRs can be assigned and that issues/PRs can only have 10 assignees at the same time.\nFor more information please see [the contributor guide](https://git.k8s.io/community/contributors/guide/#issue-assignment-in-github)", strings.Join(mu.Users, ", "), org, org)
```

Comment:
I think we should preserve repo collaborators because they can be assigned as well.

So maybe something like `members, repo collaborators and people who have commented on this issue/PR can be assigned...`?

Reply:
It's fine to me.

Just a question. Are there people who is the collaborator but not a member?

Reply:
> Are there people who is the collaborator but not a member?

Collaborators who are not members are [outside collaborators](https://help.github.com/en/articles/adding-outside-collaborators-to-repositories-in-your-organization). We don't want to have them in upstream Kubernetes, but I believe we still have some outside collaborators (we intend to fix this).

Other users of prow can have cases where they have outside collaborators so it's useful to keep this line in. :)

Reply:
OK. Thanks for your patience and quick response. :)